docs/table_design/Data_distribution.md

```diff
@@ -1,107 +1,209 @@
+
+- The first level is partitioning: Data within a table can be partitioned. Supported partitioning methods are expression partitioning, range partitioning, and list partitioning. Or you can choose not to use partitioning (the entire table is regarded as one partition).
+- The second level is bucketing: Data in a partition needs to be further distributed into smaller bucktes. Supported bucketing methods are hash and random bucketing.
+
+| **Distribution method**   | **Partitioning and bucketing method**                        | **Description**                                              |
+| ------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Random distribution       | Random bucketing                                             | The entire table is considered a partition. Data in the table is randomly distributed into different buckets. This is the default data distribution method. |
+| Hash distribution         | Hash bucketing                                               | The entire table is considered a partition. The data in the table is distributed to the corresponding bucket, which is based on the hash value of the data's bucketing key by using a hash function. |
+| Range+Random distribution | <ol><li>Expression partitioning or range partitioning </li><li>Random bucketing </li></ol> | <ol><li>The data in the table is partitioned based on the ranges of values in the partitioning columns. </li><li>The data in the partition is randomly distributed across different buckets. </li></ol> |
+| Range+Hash distribution   | <ol><li>Expression partitioning or range partitioning</li><li>Hash bucketing </li></ol> | <ol><li>The data in the table is partitioned based on the ranges of values in the partitioning columns.</li><li>The data in the partition is distributed to the corresponding bucket, which is based on the hash value of the data's bucketing key by using a hash function.</li></ol> |
+| List+Hash distribution    | <ol><li>Expression partitioning or List partitioning</li><li>Hash bucketing </li></ol> | <ol><li>The data in the table is partitioned based on the values in the partitioning columns.</li><li>The data in the partition is distributed to the corresponding bucket, which is based on the hash value of the data's bucketing key by using a hash function.</li></ol> |
+| List+Random distribution  | <ol><li>Expression partitioning or List partitioning</li><li>Random bucketing </li></ol> | <ol><li>The data in the table is partitioned based on the values in the partitioning column.</li><li>The data in athe partition is randomly distributed across different buckets.</li></ol> |
+
+- **Random distribution**
+
+  If you do not configure partitioning and bucketing methods at table creation, random distribution is used by default (currently only support Duplicate Key tables).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/0d8984f1-ad9e-426b-939f-a54c84bfff6f)
I think this sentence needs a subject.